salt/utils/network.py

```diff
@@ -1896,8 +1896,16 @@ def refresh_dns():
         pass
 
 
+@jinja_filter('connection_check')
+def connection_check(addr, port=80, safe=False, ipv6=None):
```

Comment:
I don't see the need for `connection_check` when `dns_check` does the *exact* same thing.

Reply:
@MTecknology Looks like you originally added the `connection_check` alias, would you care to comment on it's need versus `dns_check`?

Reply:
@garethgreenaway I don't recall adding it and I'm having a hard time finding a commit where I added it. Is there any chance you could point me toward it so maybe my memory can be triggered?

Reply:
https://github.com/saltstack/salt/pull/49482/commits/026a5763e31731cfb66509f3ac7eea20f87d078a

Reply:
@MTecknology ^^^

Reply:
Oh, hey friends! How have you fellas been?

Yes, now that I'm digging back into things, I do recall this; ``connection_check()`` was a "convenience alias" in [#46053](https://github.com/saltstack/salt/issues/46053#issuecomment-366277516).

If we prefer not have aliases, then I see no reason to keep it.

Reply:
Removing this function would effectively finish reverting commits 2e0ae24 and 026a576. I believe this is a good idea.